docs/modules/reference/pages/provisioning/detectors.adoc

```diff
@@ -5,6 +5,45 @@ This section describes all available built-in xref:operation:provisioning/introd
 You can configure them to allow complex detection to meet the needs of your environment.
 For information on how to extend the functionality of these detectors, see the xref:development:development.adoc#development[Development section] to contribute to the project.
 
+[[ref-detector-provisioning-common-parameters]]
+== Common configuration parameters
+
+Application- or device-specific detectors are based on a generic API that provides common configuration parameters.
+These minimum configuration parameters are available in all detectors and describe the behavior for timeouts, retries, and so on.
+
+.Optional common configuration parameters
+[options="header"]
+[cols="1,3,2"]
+|===
+| Parameter
+| Description
+| Default value
+
+| ipMatch
+| IP address to match.
+Use filters to match single addresses or address ranges.
+Substrings are matched by default (see note below this table for information on regular expression matching). +
+Note: ipMatch is a required parameter for xref:reference:provisioning/detectors/LoopDetector.adoc[LoopDetector].
+|
+
+| retries
+| Number of times detector should try to connect to database. +
+Note: this is not included by default for xref:reference:provisioning/detectors/BsfDetector.adoc[BSFDetector].
+|
+
+| serviceName
+| A custom service name to identify the service.
+|
+
+| timeout
+| Response time limit (in milliseconds). +
+Note: this is not included by default for xref:reference:provisioning/detectors/BsfDetector.adoc[BSFDetector].
+| 2000
+|===
+
+NOTE: If there is a `~` at the beginning of the parameter, the rest of the string will be used as a regular expression pattern match, otherwise the match will be a substring one.
```

Comment:
```suggestion
NOTE: If there is a `~` at the beginning of a parameter's value, the rest of the string will be handled as a regular expression pattern match, otherwise it will be processed as a substring match.
```

Reply:
```suggestion
NOTE: If there is a `~` at the beginning of a parameter's value, the rest of the string will be handled as a regular expression pattern match.
Otherwise it will be processed as a substring match.
```